### ISSUE TITLE
Just amazing

### ISSUE THREAD

**Issue by Falcosc** [nikp123/wake-on-arp#19]
I don't even know how it was possible to life without this tool.

Back then I did create custom redirect web apps which did send a magic package and redirected to the woken up endpoint.
I would never have dreamed from a solution where all kinds of requests do wake up the corresponding device just based on ARP.

One story from last week: My stupid TV did just remind me that we are still in the 90s because wake on ARP is still not a common feature 30 years later. After turning the TV off, it consumes 10W permanently because it waits for wake-up requests for:
- mirror cast
- remote file access
- remote TV access
- remote APP for smartphones

Because to be able to wait for those network requests, there needs to be a computer running all the time.

You need to turn off all 4 features to be able to get proper 0.1W Standby mode.
Guess what: somehow "turn TV on mirror cast request" was enabled again. I hate this TV so much, I just disabled this staff 1 year ago. If you don't monitor this stupid thing permanently, you never know if it really goes into standby.
It should be illegal to have all this crap turned on by default.

If we wouldn't be still in the 90s, this wouldn't be an issue and every device would build with wake on LAN in mind.

**Comment by nikp123** [comment]
Thanks, however I don't see it as a technological issue, but a social one.
How can you ensure to your technologically inept customer that their new "smart" thing is just going to work with their old equipment? I've seen many new things be disregarded as junk because they couldn't have been bothered to spend 5 minutes looking into the problem. And honestly I don't see a way around it.

But this whole thing wouldn't have been a problem if the software was open to begin with. So in my opinion it's just better to keep stuff open and only buy equipment that encourages that mentality. You don't have to go to the polar extreme "give me GPL or death" but have a sane consciousness about it.

Anyways, cheers.

**Comment by Falcosc** [comment]
Right, and in the end, 10W Idle power consumption in each home of the world is still nothing compared to all the other wastes of energy and resources. For that reason, nobody will enforce better solutions. The only enforcement we currently get (at least in EU) is the required possibility to turn these things off to avoid this behavior.

I did change my setup a bit, so my media PC is used less often. During this change, I did discover that my Soundbar and my TV are sending ARP requests to my Media PC. So I just discovered that the source IP blacklist is important in your project.

Since most of the people didn't even fix the larger energy issues in their home, nobody cares about the micro optimization we are doing here. In the end, it's just a hobby to extend what is technical possible. As soon as you set your foot into a car, all the little Computer power consumption optimizations get irrelevant ;)

But still, I just got reminded what a great idea this project of yours actually was. Strange that nobody else did think about it.

**Comment by nikp123** [comment]
No need to keep this open, closing.
